src/elements/_OcSidebarNavItem.vue

```diff
@@ -6,14 +6,10 @@
       v-on="!target ? { click: $_ocSidebarNavItem_onClick } : {}"
     >
       <oc-icon :name="icon" v-if="icon" />
-      <slot name="text">
-        <span>
-          {{ text }}
-        </span>
-      </slot>
-    </component>
-    <ul class="uk-nav-sub" v-if="$_ocSidebarNavItem_hasDefaultSlot">
       <slot name="default" />
+    </component>
+    <ul class="uk-nav-sub" v-if="$slots.subnav">
```

Comment:
I'd prefer to keep the sub navs in the default slot.
any text can go into a text slot

Reply:
Since text will be implemented more often than subnav I think it would mean more work than having it as a default slot.